### DISCUSSION TITLE
Timer for Maneki Neko

### DISCUSSION THREAD

**Discussion by virtnsk** [sunflower-land/sunflower-land#2281] [Ideas]
Is it possible to change the timer from "24 hours after shake" to "1 shake per day" like in the daily rewards?

Just reset the timer at midnight so we can shake it anytime during the day.

**Comment by ghost** [comment]
This is a brilliant idea!

**Comment by future2back** [comment]
#2212
I believe it's now working like this, right?

**Reply by future2back** [reply]
does anyone confirm?
@virtnsk you can close this discussion since @Pandanuty confirmed that its working like that.

**Reply by ghost** [reply]
Yes maneki is exactly how it works now
